src/components/Report/components/RankedResponses.js

```diff
@@ -324,6 +327,14 @@ export default class RankedResponses extends Component {
       </td>
     );
   }
+  renderCopyButton() {
+    return (
+      <CopyButton
+        buttonTitle="Click to copy the most recent responses"
```

Comment:
For other questionnaires, the hover says "Click to copy summary of responses". I think that copies the entire table for those questionnaires... but it doesn't seem to be working for me in Edge right now. 
Seems like it would be good to be consistent, but did Lydia or Bill want this to only copy the latest goals & expectations?

Reply:
@mcjustin I fixed the hover text to be consistent with other sections.  When copying responses in other section, only the latest responses are copied - that is what we discussed and agreed on before, at least that is what I thought.  I assume the behavior for goals and expectations section will be the same.